api/serializers.py

```
@@ -35,7 +35,7 @@
 class MedicationOrderSerializer(serializers.ModelSerializer):
     class Meta:
         model = MedicationOrder
-        fields = ['id', 'patient', 'encounter', 'atc', 'name', 'dose', 'frequency', 'route', 'start_date', 'end_date']
+        fields = ['id', 'patient', 'encounter', 'atc', 'name', 'dose', 'frequency', 'start_date', 'end_date']
```

Ruff (0.12.2)

Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)